presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/alluxio/AlluxioHiveMetastore.java

```diff
@@ -145,41 +143,41 @@ public void updatePartitionStatistics(String databaseName, String tableName,
     }
 
     @Override
-    public List<String> getAllTables(String databaseName)
+    public Optional<List<String>> getAllTables(String databaseName)
     {
         try {
-            return client.getAllTables(databaseName);
+            return Optional.of(client.getAllTables(databaseName));
         }
         catch (AlluxioStatusException e) {
             throw new PrestoException(HIVE_METASTORE_ERROR, e);
         }
     }
 
-    @Override
-    public List<String> getTablesWithParameter(String databaseName, String parameterKey, String parameterValue)
-    {
-        try {
-            return client.getAllTables(databaseName).stream().filter(s -> {
-                        // TODO Is there a way to do a bulk RPC?
-                        Table table = getTable(databaseName, s).orElse(null);
-                        if (table == null) {
-                            return false;
-                        }
-                        String value = table.getParameters().get(parameterKey);
-                        return value != null && value.equals(parameterValue);
-                    }
-            ).collect(Collectors.toList());
-        }
-        catch (AlluxioStatusException e) {
-            throw new PrestoException(HIVE_METASTORE_ERROR, e);
-        }
-    }
-
-    @Override
-    public List<String> getAllViews(String databaseName)
+//    @Override
```

Comment:
is it commented out because it is not needed? not in the prestodb interface? or some other reason? could you make a comment about commenting it out?

Reply:
good catch. Not required. I just removed it